.github/workflows/publish-hotfix-release.yml

```diff
@@ -0,0 +1,267 @@
+#################################################################################
+#  Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
```

Comment:
pls use specific copyrights. also the date is wrong.

Reply:
addressed in [dab1246](https://github.com/eclipse-tractusx/tractusx-edc/pull/1511/commits/dab1246f08c182190f8b4cb9ad9c19e3f0e557a2)